pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.2</version>
+    <version>1.580.1</version>
```

Comment:
Yeah, I just did not want to deal with such an old baseline when I knew it was going to be bumped up anyway.